graph/cpp/graph.cpp

```diff
@@ -469,7 +469,7 @@ int main(int argc, char *argv[]) {
 	printf("[config] -O3\n");
 #else
 	printf("[config] badness.\n");
-	return 1;
+	// return 1;
```

Comment:
Let's just get rid of this whole `#if` block.

Reply:
Fixed [87e506c](https://github.com/nicovank/LangBench/pull/4/commits/87e506ce98c3f5e2cc3a250f722da6da76ef4904)